.gitignore

```diff
@@ -72,3 +72,5 @@ env.properties
 
 !/common/src/test/resources/**
 /common/src/test/resources/.DS_Store
+
+/hercules/config.json
```

Comment:
If we ignore it, we shouldn't check it in then.

Reply:
Fixed, rename config.json to config_template.json